### DISCUSSION TITLE
Could not Save Dashboard  -A chart title is required.

### DISCUSSION THREAD

**Discussion by poulhoecherl** [datalust/seq-tickets#2259] [General]
I am setting up a Seq demo for my team, which includes how to create new Dashboards.

However I keep getting an error message: "Could not Save Dashboard - A chart title is required." whenever I try to save a new Dashboard.

[The documentation](https://docs.datalust.co/docs/dashboards) mentions "All changes made to the charts on a dashboard need to be saved using the save button; changes can be discarded with the revert button."

But I am not seeing these buttons anywhere, and I cant see an example in the docs or just searching.

Since I can't figure out how to save the chart title, I can't save the new Dashboard.

Is this bc I am using the Personal version?

![image](https://github.com/user-attachments/assets/b097173c-3c36-421e-b6bb-f0152c894e32)

**Comment by nblumhardt** [comment]
Hi @poulhoecherl,

The green bar just below the message is the title area for the chart - clicking in there will let you type a title for the chart.

The "+" (new chart) button normally sets focus here, but I think it'd be nicer for us to default to a title like "New Chart", so I'll take a look at it. Thanks for the feedback!

Best regards,
Nick
